assets/airports/cyhz.json

```diff
@@ -0,0 +1,2633 @@
+            "airlines": [
+                ["aal", 1],
+                ["aca", 29],
+                ["cfg", 2],
+                ["dal", 1],
+                ["emb", 3],
+                ["fdx", 1],
+                ["fle", 2],
+                ["fpo", 2],
+                ["kal", 2],
+                ["n/cessna", 8],
+                ["poe", 5],
+                ["pvl", 1],
+                ["qtr", 2],
+                ["ual", 1],
+                ["wen", 5],
+                ["wja", 15]
+            ]
+        },
+        {
+            "origin": "CYHZ",
+            "destination": "",
+            "category": "departure",
+            "route": "CYHZ.CYHZ4.KEBNI..MOWND",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.792,
+            "airlines": [
+                ["aal", 1],
+                ["aca", 29],
+                ["cfg", 2],
+                ["dal", 1],
+                ["emb", 3],
+                ["fdx", 1],
+                ["fle", 2],
+                ["fpo", 2],
+                ["kal", 2],
+                ["n/cessna", 8],
+                ["poe", 5],
+                ["pvl", 1],
+                ["qtr", 2],
+                ["ual", 1],
+                ["wen", 5],
+                ["wja", 15]
+            ]
+        },
+        {
+            "origin": "CYHZ",
+            "destination": "",
+            "category": "departure",
+            "route": "CYHZ.CYHZ4.NOTOP",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.792,
+            "airlines": [
+                ["aal", 1],
+                ["aca", 29],
+                ["cfg", 2],
+                ["dal", 1],
+                ["emb", 3],
+                ["fdx", 1],
+                ["fle", 2],
+                ["fpo", 2],
+                ["kal", 2],
+                ["n/cessna", 8],
+                ["poe", 5],
+                ["pvl", 1],
+                ["qtr", 2],
+                ["ual", 1],
+                ["wen", 5],
+                ["wja", 15]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "CYHZ",
+            "category": "arrival",
+            "route": "CAYLY.CABOT2.CYHZ",
+            "altitude": 12000,
+            "speed": 250,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["aal", 1],
+                ["aca", 29],
+                ["dal", 1],
+                ["emb", 3],
+                ["fdx", 1],
+                ["fle", 2],
+                ["n/cessna", 8],
+                ["poe", 5],
+                ["pvl", 1],
+                ["ual", 1],
+                ["wen", 5],
+                ["wja", 15]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "CYHZ",
+            "category": "arrival",
+            "route": "EBONY.FUNDY2.CYHZ",
+            "altitude": [24000, 18000],
```

Comment:
I'm not sure if it even makes a difference, but for the sake of consistency, the lower limit should be first, then the upper limit. Same applies to a few other spawnPatterns